setup.cfg

```diff
@@ -104,7 +105,7 @@ parentdir_prefix =
 
 [flake8]
 max-line-length = 99
-doctests = True
+doctests = False
```

Comment:
why has this changed?

Reply:
Because flake8 barfs on doctests that use variables that were provided by the `doctest_namespace` fixture, and I didn't want to rewrite the doctests that I copied from niworkflows.